packages/flutter/lib/src/cupertino/colors.dart

```diff
@@ -0,0 +1,40 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+
+class CupertinoColors {
+  CupertinoColors._();
+
+  /// iOS 10's default blue color. Used to indicate active elements such as
+  /// buttons, selected tabs and your own chat bubbles.
+  static const Color activeBlue = const Color(0xFF007AFF);
+
+  /// iOS 10's default green color. Used to indicate active accents such as
+  /// the switch in its on state and some accent buttons such as the call button
+  /// and Apple Map's 'Go' button.
+  static const Color activeGreen = const Color(0xFF4CD964);
+
+  /// Opaque white color. Used for backgrounds and fonts against dark backgrounds.
+  static const Color white = const Color(0xFFFFFFFF);
+
+  /// Opaque black color. Used for texts against light backgrounds.
+  static const Color black = const Color(0xFF000000);
+
+  /// Used in iOS 10 for light background fills such as the chat bubble background.
+  static const Color lightBackgroundGray = const Color(0xFFE5E5EA);
+
+  /// Used in iOS 10 for unselected selectables such as tab bar items in their
+  /// inactive state.
+  ///
+  /// Note it's not the same gray as disabled buttons etc.
```

Comment:
avoid the word "note" -- see style guide https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-empty-prose